### PR TITLE
Update trip permissions

### PR DESCRIPTION
**Pull request by reinbach**
[ch11571]

Disable edit permissions for Travel Focal Point user in completed state for end_date and start_date fields

### CODE REVIEW

File: src/etools/applications/t2f/helpers/permission_matrix.py [hunk 1]
```diff
@@ -36,16 +37,38 @@ def get_user_role_list(user, travel=None):
...
+            to_format[user][state] = data
+    return to_format
+
+
 def get_permission_matrix():
     permission_matrix = cache.get(PERMISSION_MATRIX_CACHE_KEY)
     if not permission_matrix:
-        path = os.path.join(
-            os.path.dirname(t2f.__file__),
-            "permission_matrix.json",
```
Comment (domdinicola) [inline src/etools/applications/t2f/helpers/permission_matrix.py]: can we get rid of this file then?

Comment (reinbach) [reply src/etools/applications/t2f/helpers/permission_matrix.py]: file has been removed

**Comment by domdinicola** [comment]
@reinbach please fix conflicts,
so we generate the matrix on the fly and we cache it right?

**Comment by reinbach** [comment]
@domdinicola  Yes

**Pull request merged**